### PR TITLE
#589 - Check vehicle damage

### PR DESCRIPTION
**Pull request by BoGuu**
Read the title and file :+1:

### CODE REVIEW

**Comment by Liam0** [comment]
Yep, works fine now we just need to add it into the scripts that need it.

**Comment by BoGuu** [comment]
@Liam0 

Be my guest, I am quite busy :dash: :+1:

**Comment by HyunsukBaek** [comment]
i patched boguu's commit, and then cannot use REPAIR menu as disable.

**Comment by Liam0** [comment]
@HyunsukBaek i just doublechecked pulled a fresh mission file from the github, works fine.

File: Altis_Life.Altis/core/functions/fn_getDamage.sqf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+*		File: fn_getDamage.sqf
+*		Author: BoGuu
+*
+*		Description:
+*		Returns true if the target is damaged, else returns false.
+*/
+
+private["_damage","_return"];
+params [
```
Comment (sargken) [inline Altis_Life.Altis/core/functions/fn_getDamage.sqf]: I don't want to use nit picky but the params should all be one line.

**Pull request merged**